src/methods/fetch-apps.js

```diff
@@ -65,6 +65,11 @@ export default (meta = {}) => {
 
   let url = '/applications?'
   if (params) {
+    for (const key in params) {
+      if (params[key] === undefined || params[key] === null) {
+        delete params[key]
+      }
+    }
     url += Object.keys(params).map(key => key + '=' + params[key]).join('&')
```

Comment:
Ok, funcionaria, mas por que não só usar o `for in` neste caso (ou só `keys.filter.map`)?
Acho desnecessário percorrer o objeto todo duas vezes..

Além disto, eu tinha sugerido verificar se é undefined, mas pensando melhor acho que é bom verificar o `typeof`, assim além de evitar o undefined/null você também evita objetos (que não podem ser concatenados à string).

```js
    for (const key in params) {
      switch (typeof params[key]) {
        case 'string':
        case 'number':
          url += `${key}=${params[key]}&`
      }
    }
```